### ISSUE TITLE
COCO Pretrain results

### ISSUE THREAD

**Issue by faheuer** [jfzhang95/pytorch-deeplab-xception#158]
I am struggling to train the network on COCO. It keeps diverging after 1 or 2 epochs, whichever learning rate or class subsamples I use. Can someone help me with this or:

Did someone train on COCO and can tell me their mIOU results and ideally individual class IOUs?
